integration/core/test_cli.py

```diff
@@ -657,11 +657,11 @@ def get_backup_url(bin, backupID):
     for x in range(RETRY_COUNTS):
         output = subprocess.check_output(cmd).strip()
         backup = json.loads(output)
-        if backup['progress'] == 100 and 'backupURL' in backup.keys():
-            rValue = backup['backupURL']
+        if backup['progress'] == 100 and 'url' in backup.keys():
```

Comment:
I think we can keep `backupURL` though since it's always used in this way. `backupstatus.url` is a bit misleading.